proposals/stage-2/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -10,11 +10,16 @@ Generally, reports to coc-escalation should refer to a report that was already s
 
 * report@openjsf.org or
 * one of the project's reporting emails 
-* explanation of why the reportee does not feel comfortable using the previous reporting emails for the initial report
+* explanation of why the reporter does not feel comfortable using the previous reporting emails for the initial report
 
-If a report to coc-esclation does not include one of those, the reportee will be asked to to provide one of those three as the 
+If a report to coc-esclation does not include one of those, the reporter will be asked to to provide one of those three as the 
 reason for reporting to coc-escalation.
 
+## Code of Conduct
+
+The OpenJS Foundation and its member projects use the Contributor Covenant v1.4.1 as its Code of Conduct. Refer to the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md) for the full text and reporting and esclation procedures. 
```

Comment:
I think that the version number is just 1.4.

```suggestion
The OpenJS Foundation and its member projects use the Contributor Covenant v1.4 as its Code of Conduct. Refer to the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md) for the full text and reporting and esclation procedures. 
```

Reply:
Note I think this is also incorrect in the CoC itself (and could be fixed as part of this PR).

Reply:
I believe it's actually 1.4.1, the versioning on the website slug / repo just isn't particularly clear. See: https://www.contributor-covenant.org/#contributor-covenant-v1-4-1

Additionally, 2.0 seems to have shipped: https://www.contributor-covenant.org/version/2/0/code_of_conduct